trail/upload/process_uploads/standardizers/moa_standardizer.py

```diff
@@ -69,16 +69,11 @@ def standardizeMetadata(self):
         return meta
 
     def standardizeWcs(self, **kwargs):
-        # no matter how hard I try, I do not understand how it would ever be
-        # possible to extract WCS data out of this header. This entire header
-        # is nonsense...
-        wcs = Wcs(
-            wcs_radius=-999.999,
-            wcs_center_x=-999.999,
-            wcs_center_y=-999.999,
-            wcs_center_z=-999.999,
-            wcs_corner_x=-999.999,
-            wcs_corner_y=-999.999,
-            wcs_corner_z=-999.999
-        )
+        # ignores any WCS information in the header and instead just sends it to astrometry.net to solve.
+        # need to add a catch that catches the errors in astrometryNetSolver
+        try:
+            header, dimX, dimY = self._astrometryNetSolver(self._kwargs['filepath'])
+            wcs = Wcs(**self._computeStandardizedWcs(header, dimX, dimY))
```

Comment:
Suggest having just one line in `try` block if feasible. Pretty sure you're catching the exception on the first line; assuming that's true, I suggest moving the second line to an `else` block (following `except`).